src/subdomains/lightning/dto/lightning-transaction.dts.ts

```diff
@@ -0,0 +1,53 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { IsDate, IsEnum } from 'class-validator';

```

Comment:
Remove all decorators from `class-transformer` and `class-validator`. These are not required on a output DTO.